sql_generators/mobile_kpi_support_metrics/templates/new_profiles.query.sql

```diff
@@ -0,0 +1,44 @@
+{{ header }}
+SELECT
+  first_seen_date,
+  normalized_channel,
+  app_name,
+  app_version,
+  country,
+  locale,
+  isp,
+  os,
+  os_version,
+  device_manufacturer,
+  is_mobile,
```

Comment:
`paid_vs_organic` has only two possible values, I think it could be worth adding here.

Reply:
my bad, I just added it to the corresponding view (the same way we did it with engagement and retention).